ckanext/canada/plugins.py

```diff
@@ -97,10 +98,12 @@ def before_update(self, context, current, resource):
         """
 
     def get_validators(self):
-        return {'canada_security_upload_type':
-                    validators.canada_security_upload_type,
-                'canada_security_upload_presence':
-                    validators.canada_security_upload_presence}
+        validators_dict = super(CanadaSecurityPlugin, self).get_validators() or {}
+        validators_dict.update({'canada_security_upload_type':
+                                    validators.canada_security_upload_type,
+                               'canada_security_upload_presence':
+                                    validators.canada_security_upload_presence})
+        return validators_dict
```

Comment:
doesn't matter in this case, but for when it does I like to avoid mutating dicts as a best-practice.
```suggestion
        return dict(
            validators_dict,
            canada_security_upload_type=validators.canada_security_upload_type,
            canada_security_upload_presence=validators.canada_security_upload_presence,
        )
```